### PR TITLE
Logout functionality

### PR DESCRIPTION
**Pull request by Pherpher089-zz**
Added logout functionality

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: groovin-music/src/components/Context/MainContext.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React, { useState } from 'react';
 
+//Main context object used in the app.
 const MainContext = React.createContext({
```
Comment (djzaragoza) [inline groovin-music/src/components/Context/MainContext.js]: nice job on building out the context

File: groovin-music/src/components/Home/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -1,7 +1,18 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import axios from 'axios';
 import './Sidebar.css';
 
+function logout() {
```
Comment (djzaragoza) [inline groovin-music/src/components/Home/Sidebar/Sidebar.js]: so cool for the user to be able to log in and out now!

**Review by djzaragoza — APPROVED** [review]
Overall a job well done!

**Review by CampbellD84 — APPROVED** [review]
So logout() is just an Axios call to the logout route? Good work implementing Context and getting logout functionality working!

**Pull request merged**